src/Host/Client/Impl/rtvs/R/overrides.R

```diff
@@ -82,3 +82,25 @@ remove.packages <- function(...) {
     utils::remove.packages(...)
     invisible(rtvs:::send_notification('!PackagesRemoved'))
 }
+
+suppress_function <- function(function_name, package_name) {
+    package_spec <- paste("package:", package_name, sep='');
+    # The message may only become visible in remote session.
+    # TODO: provide localized replacement.
+    replacement <- eval(parse(text = "function(...) { stop('Not supported') }"));
```

Comment:
I don't think you need eval here, and you definitely don't need parse. If eval is needed, you can use quote:
```r
replacement <- eval(quote(function(...) { stop('Not supported') }))
```
But I don't see the reason for eval, either. You should just be able to use the function literal directly.